macros/on-run-end/print_dbt_project_evaluator_issues.sql

```diff
@@ -0,0 +1,53 @@
+{% macro _return_list_header_rows(query) %}
+
+    {%- call statement('get_query_results', fetch_result=True,auto_begin=false) -%}
+
+        {{ query }}
+
+    {%- endcall -%}
+
+    {% set sql_results=[] %}
+
+    {%- if execute -%}
+        {% set sql_results_table = load_result('get_query_results').table %}
+        {% do sql_results.append(sql_results_table.column_names) %}
+        {% for row_data in sql_results_table.rows %}
+            {% do sql_results.append(row_data.values()) %}
+        {% endfor %}
+    {%- endif -%}
+
+    {{ return(sql_results) }}
+
+{% endmacro %}
+
+
+{% macro print_dbt_project_evaluator_issues(schema_package=None, database_package=None) %}
```

Comment:
can you explain these arguments? these are just so that a person could manually specify the db.schema location of the package models? and by default, the compiled query will just be `select * from model_name`? is that correct?

Reply:
Yes, it looks like I can't actually `ref` the models (because dbt then says that it finds dependencies it didn't find at parse time) so I allow people to mention which db/schema their models are.